packages/trusted-contribution/src/trusted-contribution.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import {
+  ChecksCreateParams,
+  PullsListFilesResponse,
+  PullsListFilesResponseItem,
+  Response,
+} from '@octokit/rest';
+
+type Conclusion =
+  | 'success'
+  | 'failure'
+  | 'neutral'
+  | 'cancelled'
+  | 'timed_out'
+  | 'action_required'
+  | undefined;
+
+// TODO: add this to a configuration file to be extended per repository.
+const TRUSTED_CONTRIBUTORS = ['renovatebot', 'release-please'];
```

Comment:
we could potentially pull this into config we load from the repo, but this seems great so far.

Reply:
That is my plan also. I wanted to get this in before expanding since it was my first bot and I know, like this, it will provide a good amount of value.

Reply:
https://github.com/googleapis/repo-automation-bots/issues/117 to track.